### ISSUE TITLE
Community Links Located under South West not London

### ISSUE THREAD

**Issue by edwardwptc** [TwinePlatform/twine-temp-dash#12]
Hi both,

Admin called from Community Links in London - on user signup their organisation is not listed under London.

I searched each Region and found them under "South West". 

Obviously they'd like to be moved back to London. 

Other organisations currently listed under "South West" but should be elsewhere:
Centre4 - Yorkshire and the Humber 
Antwerp Arms - London
Fordhall Organic Farm - West Midlands
Friends of Hardie Park - South East
Friends of Stretford Public Hall - North West
From Cheese and Grain - South East
Glamis Hall - East Midlands
Haslingden Community Link - North West
Highfields Community Association - East Midlands
Ideal for All - West Midlands
Inspired Neighbourhood - Yorkshire and the Humber
Meadow Well Connected - North East
Neston Community Youth Centre - North West
Playworks - East Midlands
Reviive CIC - West Midlands
The Florrie - North West
The Parr Sports & Community Centre - East Midlands
The Wharton Trust - North East

Sidenote: Many of the organisations listed under South West are not capitalised and that is not ideal - do we know why that is and is it easy to fix?

**Comment by astroash** [comment]
@edwardwptc I started changing these on the db, but on second thought the easiest way to edit these accounts is using the twine admin dashboard. Accounts can be editted by going to Organisation, then clicking the edit button next to the one you want to change. This will take you to a form where you can retype organisations names and change their regions.

The current names that are displayed are saved as users have typed them on sign up. Usually we do not automate capitalisation as this can cause issues for certain names that don't follow the standard format eg McCarthy, Hills and Charter ect. If you would like for names to be automatically capitalised we can talk about how we implement this.

**Comment by edwardwptc** [comment]
Great, I'll do this. Thanks. Will mark as closed once complete. Happy to be able to do some work rather than get you to do it 👍

**Comment by edwardwptc** [comment]
Right @astroash  these are all done.

I've attempted to use the same dashboard > Edit details to capitalise people so that the names look more professional, but these edits don't seem to have been saved (region changes were saved). Is there some way I can edit these?

**Comment by astroash** [comment]
**Issue:**
- `twine-temp-dash` is unable to update cb's names using `PUT /community-business/:id`

**Trouble shooting steps taken :**
- tests for both the route and model method `Cb.update` are able to update name
- checked db to confirm row is not updated
- `Cb.update` correctly separates `orgChangeset` & `cbChangeset`s
- transaction for `orgChangeset` returns 1 row changed. When using `.return('*')` the correct cb row is returned however the new name is not given

The error appears to be happening here:

https://github.com/TwinePlatform/twine-api/blob/e65ef3594b8848995664e973ad32b93d58173cde/src/models/community_business.ts#L243-L249

It is unclear if this is an bug with `knex` or L247

**Comment by astroash** [comment]
@eliascodes I would appreciate you looking at this when you have some time.

**Comment by eliasmalik** [comment]
Will take a look now.

**Comment by eliasmalik** [comment]
I think this is because we're using the `CITEXT` data type. It looks like postgres is comparing the value we're trying to set to the value that's already there, and if they match, it doesn't actually do the update.

So if we were using a `TEXT` data type, it would compare, for example, `Twine` to `twine`, notice a difference, and then update the value. But using `CITEXT`, there's an implicit application of `LOWER()`, so it is actually comparing `LOWER('Twine')` to `LOWER('twine')`, which is the same.

Perhaps unhelpfully, in both cases, the query returns `UPDATE 1`, telling the client that a row has been updated.

I came to this conclusion by trying to update org names: 
```sql
UPDATE organisation SET organisation_name='Extra workspace 150' WHERE organisation_id=X;
```
This returns `UPDATE 1` but the name remains `Extra Workspace 150`. But if we change anything else, then the update "works":
```sql
UPDATE organisation SET organisation_name='Extra workspace 151' WHERE organisation_id=X;
```
Now the name is `Extra workspace 151`.

Will try to think of a way forward.
